### DISCUSSION TITLE
Error while syncing with One Drive Personal

### DISCUSSION THREAD

**Discussion by tangram9** [remotely-save/remotely-save#889] [Q&A]
Hi, I'm getting the following error while syncing files to my One Driver account. 

i'm using an iPhone and a PC to sync with my One Drive account. 

Error: 

We meet file / folder and do not know how to deal with it: ParentPath = "/drive/root/Apps/Graph 1, SelfName='Obsidian" 

remotely-save-1731478353307: abort sync, triggerSource=manual, error while getting_remote_files_list 

I had been able to sync for a month, then suddenly i started receiving this error one day and unable to sync thereafter. 

I'm able to sync from my PC but not from my iPhone. 

Kindly inform of any resolution for this error. 

I have checked the file path in One drive. I have a file named remotely-save under Apps folder. I have checked to see any hidden folders with the name Graph 1 but there are none.

**Comment by marcosllinares** [comment]
Same error here

**Comment by imbaxl** [comment]
I've changed folder name in completely the same, i mean 
Apps\remotely-save\MathBook
 into 
Apps\remotely-save\Mathbook
finally it worked.
